common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -404,6 +405,9 @@ def test_video_with_youtube_blocked(self):
 
         self.assertTrue(self.video.is_video_rendered('html5'))
 
+        # The video should only be loaded once
+        self.assertEqual(len(self.video.q(css='video')), 1)
```

Comment:
Ah, looks like even though this test claimed it was checking the case when the request to fetch the Youtube API returned an error after the wait timeout, the error response was being returned before the wait timeout.

Can you please add a test for the case when the error is returned before 1.5 seconds? Then we will have coverage for all four cases.